modules/sd-terraform-prereqs.adoc

```diff
@@ -0,0 +1,120 @@
+                "iam:UpdateAssumeRolePolicy",
+                "secretsmanager:DescribeSecret",
+                "iam:ListRoleTags",
+                "secretsmanager:PutSecretValue",
+                "secretsmanager:CreateSecret",
+                "iam:TagRole",
+                "secretsmanager:DeleteSecret",
+                "iam:UpdateOpenIDConnectProviderThumbprint",
+                "iam:DeletePolicy",
+                "iam:CreateRole",
+                "iam:AttachRolePolicy",
+                "iam:ListInstanceProfilesForRole",
+                "secretsmanager:GetSecretValue",
+                "iam:DetachRolePolicy",
+                "iam:ListAttachedRolePolicies",
+                "iam:ListPolicyTags",
+                "iam:ListRolePolicies",
+                "iam:DeleteOpenIDConnectProvider",
+                "iam:DeleteInstanceProfile",
+                "iam:GetRole",
+                "iam:GetPolicy",
+                "iam:ListEntitiesForPolicy",
+                "iam:DeleteRole",
+                "iam:TagPolicy",
+                "iam:CreateOpenIDConnectProvider",
+                "iam:CreatePolicy",
+                "secretsmanager:GetResourcePolicy",
+                "iam:ListPolicyVersions",
+                "iam:UpdateRole",
+                "iam:GetOpenIDConnectProvider",
+                "iam:TagOpenIDConnectProvider",
+                "secretsmanager:TagResource"
+            ],
+            "Resource": [
+                "arn:aws:secretsmanager:*:<ACCOUNT_ID>:secret:*",
+                "arn:aws:iam::<ACCOUNT_ID>:instance-profile/*",
+                "arn:aws:iam::<ACCOUNT_ID>:role/*",
+                "arn:aws:iam::<ACCOUNT_ID>:oidc-provider/*",
+                "arn:aws:iam::<ACCOUNT_ID>:policy/*"
+            ]
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:AbortMultipartUpload",
+                "iam:ListRoles",
+                "s3:ListBucket",
+                "sts:AssumeRoleWithWebIdentity",
+                "s3:DeleteObject",
+                "s3:GetBucketPolicy",
+                "s3:GetObjectVersion",
+                "s3:ListMultipartUploadParts",
+                "s3:PutInventoryConfiguration",
+                "s3:CreateBucket",
+                "s3:PutBucketPolicy",
+                "s3:DeleteBucketPolicy"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+====
+
+* **Optional**: A configured `terraform.tfvars` file.
++
+A `terraform.tfvars` file is a definition sheet for all of your variables. This method allows you to reference a file that is safely stored due to the sensitive nature of some variable values. You can create multiple `terraform.tfvars` files with different variable values. See link:https://developer.hashicorp.com/terraform/language/values/variables[Hashicorp's documentation] for information on input variables.
```

Comment:
1. Maybe: "A `terraform.tfvars` file is a definition sheet that allows you to safely store and reference variables. You can create..."
2. Do we need to say "input variables" in the last line when we just called them "variables" the rest of the time?